pkg/apis/policy/validation/validation.go

```diff
@@ -310,11 +303,14 @@ func IsValidSysctlPattern(name string) bool {
 // validatePodSecurityPolicySysctls validates the sysctls fields of PodSecurityPolicy.
 func validatePodSecurityPolicySysctls(fldPath *field.Path, sysctls []string) field.ErrorList {
 	allErrs := field.ErrorList{}
+	if sysctls == nil {
```

Comment:
if the feature gate is disabled, you should validate they are not set.
if the feature gate is enabled, you should do normal validation.

Reply:
`validatePodSecurityPolicySysctls` is run on the apiserver side where the feature gate functionality is not available